libexec/rtld-elf/riscv/rtld_start.S

```diff
@@ -114,7 +114,8 @@ ENTRY(_rtld_bind_start)
 	unimp
 #else /* defined(__CHERI_PURE_CAPABILITY__) */
 	/* Save the arguments and ra */
-	addi	sp, sp, -(8 * 17)
+        /* We require 17 dwords, but the stack must be aligned to 16-bytes */
```

Comment:
```suggestion
	/* We require 17 dwords, but the stack must be aligned to 16-bytes */
```

Reply:
I wonder if you could use the STACKALIGN macros here somehow?  I guess not if they are now using __builtin_align instead of simple C-like roundup2().  Would be nice if it could be something like `addi    sp, sp, -STACKALIGN(8 * 17)`.